src/network/networkd-network.c

```diff
@@ -541,6 +542,17 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
         if (stat(filename, &stats) >= 0)
                 network->timestamp = timespec_load(&stats.st_mtim);
 
+        char **f;
+        STRV_FOREACH(f, dropins) {
+                usec_t t;
+
+                if (stat(*f, &stats) < 0)
+                        continue;
```

Comment:
Hmm, that's not correct either. I think `network->timestamp` should be reset to 0 here and `break`.

Reply:
I see. Updated.